lambda-buffers-compiler/src/LambdaBuffers/Compiler/ProtoCompat/Types.hs

```diff
@@ -6,6 +6,8 @@
 {-# OPTIONS_GHC -fconstraint-solver-iterations=0 #-}
 
 module LambdaBuffers.Compiler.ProtoCompat.Types (
+  foreignRef2LocalRef,
+  localRef2ForeignRef,
```

Comment:
Move this to the module where it's called. Keep this module clean as much as possible.

Also, add a `TODO(cstml): Purge Arbitrary instances and make necessary generators in the test suite`

Reply:
`localRef2Foreign...` is called in two places. Making a module just for this seems overkill.